src/Cms/App.php

```diff
@@ -924,7 +924,8 @@ public function markdown(string $text = null, $inline = null): string
             (array)$options
         );
 
-        return ($this->component('markdown'))($this, $text, $options);
+        $inline = $options['inline'] ?? false;
```

Comment:
Could you add an inline `/// TODO:` comment please that we deprecate passing the parameter in 3.7 and remove passing it in 3.8? Thanks.